<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Focuser</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(55, 55, 55);</string>
  </property>
  <widget class="QWidget" name="currentWindow">
   <widget class="QStackedWidget" name="pageSelect">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>331</width>
      <height>451</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="btnConnectClient">
      <property name="geometry">
       <rect>
        <x>128</x>
        <y>110</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: forestGreen;</string>
      </property>
      <property name="text">
       <string>Conectar</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtClientIp">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;</string>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>55</red>
            <green>55</green>
            <blue>55</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>55</red>
            <green>55</green>
            <blue>55</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>55</red>
            <green>55</green>
            <blue>55</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="PlaceholderText">
          <brush brushstyle="SolidPattern">
           <color alpha="128">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>55</red>
            <green>55</green>
            <blue>55</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>55</red>
            <green>55</green>
            <blue>55</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>55</red>
            <green>55</green>
            <blue>55</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="PlaceholderText">
          <brush brushstyle="SolidPattern">
           <color alpha="128">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>120</red>
            <green>120</green>
            <blue>120</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>55</red>
            <green>55</green>
            <blue>55</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>120</red>
            <green>120</green>
            <blue>120</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>55</red>
            <green>55</green>
            <blue>55</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>55</red>
            <green>55</green>
            <blue>55</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="PlaceholderText">
          <brush brushstyle="SolidPattern">
           <color alpha="128">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Server IP address</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="btnDown">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>40</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="toolTip">
       <string>Focus Down</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(222, 222, 222);</string>
      </property>
      <property name="text">
       <string>IN</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../BlackStar/src/assets/icons/down.png</normaloff>../../../BlackStar/src/assets/icons/down.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="btnHome">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>230</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: forestGreen;</string>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMove">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>230</y>
        <width>40</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="toolTip">
       <string>Go to</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(222, 222, 222);</string>
      </property>
      <property name="text">
       <string>GO</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../BlackStar/src/assets/icons/start.ico</normaloff>../../../BlackStar/src/assets/icons/start.ico</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="btnUp">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>40</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="toolTip">
       <string>Focus Up</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(222, 222, 222);</string>
      </property>
      <property name="text">
       <string>OUT</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../BlackStar/src/assets/icons/ups.png</normaloff>../../../BlackStar/src/assets/icons/ups.png</iconset>
      </property>
     </widget>
     <widget class="QTextEdit" name="txtStatus">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>311</width>
        <height>121</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>191</width>
        <height>161</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="title">
       <string>Status</string>
      </property>
      <widget class="QLabel" name="statMov_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>60</y>
         <width>40</width>
         <height>20</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Status</string>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: indianred</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Moving</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Connection</string>
       </property>
      </widget>
      <widget class="QLabel" name="statConn_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>30</y>
         <width>40</width>
         <height>20</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Status</string>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: indianred</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Initiating</string>
       </property>
      </widget>
      <widget class="QLabel" name="statInit_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>90</y>
         <width>40</width>
         <height>20</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Status</string>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: indianred</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Busy</string>
       </property>
      </widget>
      <widget class="QLabel" name="statBusy_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>120</y>
         <width>40</width>
         <height>20</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Status</string>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: indianred</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>100</y>
        <width>41</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../img/logoLna.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnUpdateStatus">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>290</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: forestGreen;</string>
      </property>
      <property name="text">
       <string>Status</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnConnect">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>260</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: forestGreen;</string>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnHalt">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: Brown;</string>
      </property>
      <property name="text">
       <string>Halt</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="BarFocuser">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>51</width>
        <height>191</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar::chunk {background-color: rgb(26, 26, 26)}
QProgressBar { color: indianred; }</string>
      </property>
      <property name="minimum">
       <number>-10</number>
      </property>
      <property name="maximum">
       <number>50700</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
      <property name="textVisible">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="textDirection">
       <enum>QProgressBar::TopToBottom</enum>
      </property>
      <property name="format">
       <string>%v</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtMov">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>61</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri Light</family>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
background-color: lightgrey;
border: 1px solid rgb(2,2,2);
border-radius: 2px;

}</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
